cr8/bench_spec.py

```diff
@@ -2,6 +2,11 @@
 import os
 import json
 import toml
+import pathlib
+
+
+def file_name(full_path):
+    return pathlib.Path(full_path).name
```

Comment:
not really. I was just amazed by `pathlib` and forgot about `basename` :D 
`basename()` is probably better!